cf-check/backup.c

```diff
@@ -0,0 +1,198 @@
+#include <platform.h>
+#include <backup.h>
+
+#if defined (__MINGW32__)
+
+int backup_main(int argc, char **argv)
+{
+    printf("backup not supported on Windows\n");
+    return 1;
+}
+
+#elif ! defined (LMDB)
+
+int backup_main(int argc, char **argv)
+{
+    printf("backup only implemented for LMDB.\n");
+    return 1;
+}
+
+#else
+
+#include <stdio.h>
+
+bool copy_file(const char *src, const char *dst)
+{
+    assert(src != NULL);
+    assert(dst != NULL);
+
+    printf("Copying: '%s' -> '%s'\n", src, dst);
+    FILE *in = fopen(src, "r");
+    if (in == NULL)
```

Comment:
I see you handle the return of false elsewhere with an error message but would you want to print out the specific reason here?

Reply:
Yea, I can add messages to these.